awx/main/management/commands/export_custom_scripts.py

```diff
@@ -0,0 +1,36 @@
+import tempfile
+import tarfile
+import stat
+import os
+
+from awx.main.models.inventory import CustomInventoryScript
+
+from django.core.management.base import BaseCommand
+from django.utils.text import slugify
+
+
+class Command(BaseCommand):
+
+    help = 'Export custom inventory scripts into a tarfile.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--filename', dest='filename', type=str, default='custom_scripts.tar', help='Filename of the output tar file')
+
+    def handle(self, **options):
+        tar_filename = options.get('filename')
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            with tarfile.open(tar_filename, "w") as tar:
+
+                for cis in CustomInventoryScript.objects.all():
+                    # naming convention similar to project paths
+                    slug_name = slugify(str(cis.name)).replace(u'-', u'_')
+                    script_filename = u'_%d__%s' % (int(cis.pk), slug_name)
+                    script_path = os.path.join(tmpdirname, script_filename)
+
+                    with open(script_path, 'w') as f:
+                        f.write(cis.script)
+                    os.chmod(script_path, stat.S_IRWXU)
```

Comment:
@wenottingham Important bit here, this was modeled after your suggestions. But `oct(stat.S_IRWXU) == 0o700`, as opposed to 600, because we need the executable bit set.

What this looks like in the test container:

```
$ tar -tvf my_scripts.tar 
-rwx------ awx/root        912 2021-04-19 12:57 _17__inventory_script_bidstory
...
```